catalogs/catalog-hive/src/main/java/com/datastrato/gravitino/catalog/hive/HiveCatalogOperations.java

```diff
@@ -283,8 +283,14 @@ public HiveSchema alterSchema(NameIdentifier ident, SchemaChange... changes)
           ident.name(),
           properties.keySet());
 
+      boolean commentUpdated = false;
+      String comment = null;
       for (SchemaChange change : changes) {
-        if (change instanceof SchemaChange.SetProperty) {
+        if (change instanceof SchemaChange.UpdateComment) {
+          commentUpdated = true;
+          comment = ((SchemaChange.UpdateComment) change).getNewComment();
+          properties.put(COMMENT, comment);
```

Comment:
seems there're no need to put `COMMENT` in properties since we could store schema comments in the description.
```
        alteredDatabase.setDescription(comment);
```